docs/guide/clustering.md

```diff
@@ -15,32 +15,93 @@ is up to you and your performance needs.
 
 The minimal requirements are:
 
-- Shared HTTP cache (using Varnish)
-- Shared persistence cache and sessions (using Memcached or Redis)
+- [Shared HTTP cache (using Varnish)](http_cache.md#using-varnish)
+- [Shared persistence cache](#shared-persistence-cache) and [sessions](#shared-sessions) (using Redis or Memcached)
 - Shared database (using MySQL/MariaDB)
-- Shared filesystem (using NFS, or S3)
+- [Shared binary files](#shared-binary-files) (using NFS, or S3)
 
 For further details on requirements, see [Requirements page](../getting_started/requirements.md).
 
-!!! note
+It is also recommended to use:
 
-    Memcached must not be bound to the local address if clusters are in use, or user logins will fail.
-    To avoid this, in `/etc/memcached.conf` take a look under `# Specify which IP address to listen on. The default is to listen on all IP addresses`
+- [Solr](solr.md) for better search and performance
+- a CDN for improved performance and faster ping time worldwide
```

Comment:
We can add sub point about us having native support for Fastly in EE as HTTP Cache, _and_ CDN.